packages/opentelemetry-plugin-grpc-js/src/server/patchServer.ts

```diff
@@ -152,3 +182,27 @@ function handleServerFunction<RequestType, ResponseType>(
       break;
   }
 }
+
+/**
+ * Does not patch any callbacks or EventEmitters to omit tracing on requests
+ * that should not be traced.
+ */
+function handleUntracedServerFunction<RequestType, ResponseType>(
+  type: string,
+  originalFunc: HandleCall<RequestType, ResponseType>,
+  call: ServerCallWithMeta<RequestType, ResponseType>,
+  callback: SendUnaryDataCallback<unknown>
+): void {
```

Comment:
Are you sure the return type of `originalFunc` will always be `void` ?

Reply:
According to the grpc-js [types](https://github.com/grpc/grpc-node/blob/f08d3aefd0c63324f26fb511113d59ad755ce72e/packages/grpc-js/src/server-call.ts#L277-L310), HandleCall is always going to be a void function